terraform/modules/len/lambdas/len.py

```diff
@@ -90,6 +76,83 @@ def post_event(auth_token: str):
             "Content-Length": len(event_request_data)
         }
     )
-    logger.info(f"## Posting death certificate {death_certificate}")
+    logger.info(f"## Posting death certificate {death_registration_id}")
     request.urlopen(event_request).read()
-    logger.info(f"## Successfully posted death certificate {death_certificate}")
+    logger.info(f"## Successfully posted death certificate {death_registration_id}")
+
+
+def create_death_registration():
+    connection = psycopg2.connect(user=lev_username, password=lev_password, database=lev_db_name, host=lev_host)
+    cursor = connection.cursor()
+    cursor.execute("select nextval('death_registration_ids')")
+    death_registration_id = cursor.fetchone()
+
+    cursor.execute("""
+        INSERT INTO death_registration_v1 (id, data, forenames, surname, date_of_birth, date_of_death)
+        VALUES(%(id)s, %(data)s, %(forenames)s, %(surname)s, %(date_of_birth)s, %(date_of_death)s)
+        """,
+                   generate_death_registration(death_registration_id))
+
+    return death_registration_id
+
+
+def generate_death_registration(death_registration_id: int):
+    forenames = forenames_options[randint(0, len(forenames_options) - 1)]
+    surname = surname_options[randint(0, len(surname_options) - 1)]
+    date_of_birth = date_of_birth_options[randint(0, len(date_of_birth_options) - 1)]
+    date_of_death = date_of_death_options[randint(0, len(date_of_death_options) - 1)]
+
+    data = {
+               "id": death_registration_id,
+               "date": "2014-10-10",
+               "entryNumber": death_registration_id,
+               "registrar": {
+                   "signature": "A. Registrar",
+                   "designation": "Registrar",
+                   "subdistrict": "Test Subdistrict",
+                   "district": "Test District",
+                   "administrativeArea": "Reading"
+               },
+               "informant": {
+                   "forenames": "Informy",
+                   "surname": "McInformface",
+                   "address": "666 Inform House, 6 Inform street, Informington, Informshire",
+                   "qualification": "Life-long nemesis, Present at death",
+                   "signature": "I. McInformface"
+               },
```

Comment:
This gave me a chuckle, but I think it probably ought to be slightly more boring 😞 I know this was copied from the Home Office, but still...

Reply:
Yeah I was wondering if it should be more 'normal' hahaha